changelog.txt

```diff
@@ -34,6 +34,7 @@ Fixed
 +++++
 
  - Attempting to create a linked view for a Project on Windows now raises an informative error message.
+ - Project configuration is initialized using ConfigObj, allowing the configuration to include commas and special characters (#251, #252).
```

Comment:
Should we start enforcing that changelog entries include both issue and PR numbers? Just for future reference when I review other PRs.

Reply:
I stumbled upon this comment and went ahead and updated the pull request template: https://github.com/glotzerlab/signac/blob/master/.github/PULL_REQUEST_TEMPLATE.md

I used committer privileges to just merge this. Feel free to revert if you disagree.

Reply:
Good idea, @csadorf. Resolving this thread.